mmaction/apis/train.py

```diff
@@ -33,19 +36,48 @@ def train_model(model,
 
     # prepare data loaders
     dataset = dataset if isinstance(dataset, (list, tuple)) else [dataset]
-    dataloader_setting = dict(
-        videos_per_gpu=cfg.data.get('videos_per_gpu', 2),
-        workers_per_gpu=cfg.data.get('workers_per_gpu', 0),
-        # cfg.gpus will be ignored if distributed
-        num_gpus=len(cfg.gpu_ids),
-        dist=distributed,
-        seed=cfg.seed)
-    dataloader_setting = dict(dataloader_setting,
-                              **cfg.data.get('train_dataloader', {}))
-
-    data_loaders = [
-        build_dataloader(ds, **dataloader_setting) for ds in dataset
-    ]
+
+    if cfg.omnisource:
+        # The option can override videos_per_gpu
+        train_ratio = cfg.data.get('train_ratio', None)
+        omni_videos_per_gpu = cfg.data.get('omni_videos_per_gpu', None)
+        dataloader_setting_tmpl = dict(
+            videos_per_gpu=cfg.data.get('videos_per_gpu', {}),
+            workers_per_gpu=cfg.data.get('workers_per_gpu', {}),
```

Comment:
These two lines are not correct. (`{}` can not be set as default value for these two)